spec/controllers/bills_controller_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+describe BillsController do
+
+  before(:each) { @bill = FactoryGirl.create(:bill) }
+
+  describe 'GET #index' do
+    before { get :index }
+
+    it { is_expected.to respond_with :ok }
+    it { is_expected.to render_template :index }
+    it { expect(@bill).to_not be nil }
+  end
+
+  describe "GET #new" do
+    before { get :new }
+
+    it { is_expected.to respond_with :ok }
+    it { is_expected.to render_template :new }
+  end
+
+  describe "POST #create" do
+    let(:user) { FactoryGirl.create(:user) }
+
+    it "should create bill" do
+      sign_in user
+      bill = FactoryGirl.attributes_for(:bill).merge(user_id: user.id)
+      assert_difference('Bill.count') { post :create, bill: bill }
+    end
+  end
+
+  describe "GET #edit" do
+    before { get :edit, id: @bill }
+
+    it { is_expected.to respond_with :ok }
+    it { is_expected.to render_template :edit }
+  end
+
+  describe "POST #update" do
+    before do
+      patch(:update,id: @bill,
+        bill: {
+          amount: @bill.amount,
+          bill_received: @bill.bill_received,
+          user_id: @bill.user_id
```

Comment:
I added it when `assert_difference` did not work in the test. It might be the case that this module is included by default inthe mini-test environment but not in Rspec.